Java/README.md

```diff
@@ -333,6 +333,7 @@ _add list here_
 - [Tower of Hanoi problem](other/TowerOfHanoi.java)
 - [Unique 3](other/UniqueIII.java)
 - [LastdigitofnthFibonacci](other/LastDigitFibonacci.java)
+- [MaxMinSum](other/max_min_sum.java)
```

Comment:
make it readable and understandable

Reply:
I have done all the requested changes, kindly look into that, thank you.